app/2.3.x/external-plugins.md

```diff
@@ -0,0 +1,248 @@
+---
+title: External Plugins Support
+---
+
+## Introduction
+
+External plugins are those that run on a process separate from {{site.base_gateway}} itself,
+enabling the use of any programming language for which an appropriate
+plugin server is available.
+
+Each plugin server hosts one or more plugins and communicates with the
+main {{site.base_gateway}} process through Unix sockets.  If so configured, {{site.base_gateway}} can manage
+those processes, starting, restarting and stopping as necessary.
+
+{{site.base_gateway}} currently maintains a Go language plugin server,
+[go-pluginserver] and the corresponding PDK library
+package, [go-pdk].
+
+## Kong Gateway plugin server configuration
+
+The `pluginserver_names` property is a comma-separated list of names, one
+for each plugin server process.  These names are used to group each process'
+properties and to annotate log entries.
+
+For each name, other properties can be defined:
+
+Property | Description | Default
+---------|-------------|--------
+`pluginserver_<NAME>_socket` | Unix socket path | `/usr/local/kong/<NAME>.socket`
+`pluginserver_<NAME>_start_cmd` | Command to start the plugin server process | `/usr/local/bin/<NAME>`
+`pluginserver_<NAME>_query_cmd` | Command to dump available plugins' info | `/usr/local/bin/query_<NAME>`
+
+
+For example, you could set Go and Python plugins like this:
```

Comment:
Is python supported? If not, we should add a note.

Reply:
done (with a bit of https://www.smbc-comics.com/comic/language)